### DISCUSSION TITLE
part-ii/2019-100/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#304] [Question comments]
Part II 2019 Number Theory: Paper 3, Section I, I 

https://questions.tripos.org/part-ii/2019-100/

**Comment by callumhobbis** [comment]
<details>
<summary>Sketch solution</summary>

This is all bookwork, with a method given in lectures at the end. I got $(1,1,4)$, $(2,1,2)$, and $(3,3,2)$ as my BQFs.

</details>
